chall01/rkyttala.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/local/bin/python3
+
+# import sys for reading program arguments and regular expressions for filtering away non-ascii
+import sys, re
+
+def encoder(encodable):
+    # morse letters in order from a to z, \n as the separator
+    rawMorse = """.-
+                -... 
+                -.-. 
+                -.. 
+                . 
+                ..-. 
+                --. 
+                .... 
+                .. 
+                .--- 
+                -.- 
+                .-.. 
+                -- 
+                -. 
+                --- 
+                .--. 
+                --.- 
+                .-. 
+                ... 
+                - 
+                ..- 
+                ...- 
+                .-- 
+                -..- 
+                -.-- 
+                --.."""
+
+    rawAlphabet = "a b c d e f g h i j k l m n o p q r s t u v w x y z"
+    alphabet = rawAlphabet.split()
```

Comment:
If you're going to split both strings anyway, you should declare them both as arrays directly!